### PR TITLE
Define a common Inventory type for reuse across modules

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

File: sphinx/util/typing.py [hunk 1]
```diff
@@ -28,3 +28,5 @@
 
 # title getter functions for enumerable nodes (see sphinx.domains.std)
 TitleGetter = Callable[[nodes.Node], text_type]
+
+Inventory = Dict[str, Dict[str, Tuple[str, str, str, str]]]
```
Comment (jdufresne) [inline sphinx/util/typing.py:32]: Let me know if you'd like a comment above this line like the other types. I'm not sure what that text should be.

If the order of these definitions are important, let me know I can place this on whatever line you think is best.

Comment (tk0miya) [reply sphinx/util/typing.py:32]: Please note `# inventory data on memory`.

Comment (tk0miya) [reply sphinx/util/typing.py:32]: I'll do it after merge.

**Review by tk0miya — APPROVED** [review]

**Comment by tk0miya** [comment]
Merged. Thank you always for your hard work!

**Pull request merged**